ansible_bender/builders/buildah_builder.py

```diff
@@ -347,6 +364,7 @@ def sanity_check(self):
         run_cmd(["podman", "version"], log_stderr=True, log_output=True)
         logger.debug("checking that buildah command works")
         run_cmd(["buildah", "version"], log_stderr=True, log_output=True)
+        buildah_run_cmd(self.build.base_image, "test", ["true"], log_stderr=True)
```

Comment:
@TomasTomecek Please suggest me a better way to handle this, as I had to keep a particular static container name. As buildah does not has any direct way to run commands using container images

Reply:
this could be a good inspiration for the temporary name: https://github.com/ansible-community/ansible-bender/commit/4cae025bc50273e42aed26841359507b984338c0#diff-e820b83f0aadd2f504a891e621c3cfddR251-R254